CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.6.1
+
+## 🐞 Bug Fixes
+* Fix style and expression validation error messages in the browser ([#9073](https://github.com/mapbox/mapbox-gl-js/pull/9073))
```

Comment:
Maybe change this to:
`Fix style validation error message's not being displayed`
since "in the browser" is kind off redundant.